src/esm_runscripts/filelists.py

```diff
@@ -1276,19 +1276,19 @@ def avoid_overwriting(config, source, target):
         if filecmp.cmp(source, target):
             return target
 
+        warning_function = user_error
         date_stamped_target = f"{target}_{config['general']['run_datestamp']}"
         if os.path.isfile(date_stamped_target):
             if filecmp.cmp(source, date_stamped_target):
                 logger.info(
                     f"File {date_stamped_target} already exists and is identical to "
                     f"the source ({source}). Skipping copying"
                 )
-            elif config["general"]["force_overwrite_in_file_movements"]:
+                return date_stamped_target
+            if config["general"]["force_overwrite_in_file_movements"]:
                 os.remove(date_stamped_target)
+                # Change to user_note, the default was user_error (above)
                 warning_function = user_note
-            else:
-                # This will exit(1) (default in configs/defaults/general.yaml)
-                warning_function = user_error
```

Comment:
I think the behavior should be:
1. If force_overwrite -> the user specified to overwrite so we only through a user_error
2. If no force_overwrite -> unless the user is rerunning some simulation, this is a really strange scenario that can lead to data loss and consequently we want to through an error.

Reply:
> If force_overwrite -> the user specified to overwrite so we only throw a user_error
You mean `user_note`, right?

> If no force_overwrite -> unless the user is rerunning some simulation, this is a really strange scenario that can lead to data loss and consequently we want to through an error.

The current implementation does this, I think...? Further up, the default `warning_function` is `user_error`. So, it is "unless the user says to allow overwrite, throw an error"

Reply:
I mean that these lines should be changed to:

```python
            if config["general"]["force_overwrite_in_file_movements"]:
                os.remove(date_stamped_target)
                # Change to user_note, the default was user_error (above)
                warning_function = user_note
            else:
                # This will exit(1) (default in configs/defaults/general.yaml)
                warning_function = user_error
```

Reply:
Applied the changes suggested by me